docs/tasks/administer-cluster/storage-object-in-use-protection.md

```diff
@@ -6,7 +6,6 @@ title: Storage Object in Use Protection
 ---
 
 {% capture overview %}
-{% assign for_k8s_version="v1.10" %}{% include feature-state-beta.md %}
 
 Persistent volume claims (PVCs) that are in active use by a pod and persistent volumes (PVs) that are bound to PVCs can be protected from pre-mature removal.
```

Comment:
As long as we're making making changes to the file anyway:

s/pre-mature/premature

Reply:
@Bradamant3 I added feature state code. Is it what you were looking for?

I intentionally skipped K8s 1.9, because `Storage Object In Use` feature didn't exist in K8s 1.9. The `Storage Object In Use` feature went directly to beta.